.github/CODEOWNERS

```diff
@@ -30,9 +30,11 @@ Cargo.* @wasmCloud/host-maintainers
 # shared crates
 crates/core @wasmCloud/org-maintainers
 crates/opentelemetry-nats @thomastaylor312 @connorsmith256
+crates/provider-archive @wasmCloud/capability-provider-maintainers
 crates/provider-sdk @thomastaylor312 @connorsmith256
 crates/provider-wit-bindgen @thomastaylor312 @vados-cosmonic
 crates/tracing @thomastaylor312 @connorsmith256
+crates/wascap @thomastaylor312 @connorsmith256 @brooksmtownsend
```

Comment:
Let's add @autodidaddict to this. He's on the [existing CODEOWNERS file](https://github.com/wasmCloud/wascap/blob/main/.github/CODEOWNERS)